src/commands/issuescomment.ts

```diff
@@ -0,0 +1,56 @@
+import * as core from '@actions/core'
+import * as utils from '../utils'
+import CommandContext from './context'
+
+/**
+ * Get a list of issues from the current issue and post a comment
+ *
+ * @param {CommandContext} context context for this command execution
+ * @param {args} list of arguments
+ */
+export default async function issuescomment(
+  {owner, repo, issueNumber, issueBody, basepath}: CommandContext,
+  ...args: string[]
+): Promise<void> {
+  const message: string = args.length > 1 ? args.join(' ') : args[0]
+
+  // a little debuging info
+  core.debug(message)
+  core.debug(basepath)
+  core.debug(`/issuescomment working with ${message}`)
+  core.debug(`issue body -> ${issueBody}`)
+  const gitURLs: string[] = utils.parseExtraArgs(issueBody, 'issuescomment')
+  const filterNWO: string[] = []
+  for (const url of gitURLs) {
+    // get list of nwo's to filter on
+    const nwo: utils.NWO = utils.getNWO(url)
+    filterNWO.push(`${nwo.owner}/${nwo.name}`)
+  }
+  // eslint-disable-next-line @typescript-eslint/camelcase
+  const linkedIssues: string[] = await utils.getLinkedIssues({owner, repo, issue_number: issueNumber}, {nwo: filterNWO})
+  core.debug(`LinkedIssues -> ${linkedIssues}`)
+
+  // for each issue we need to create a comment
+  for (const url of utils.unique(linkedIssues)) {
```

Comment:
This is the only place where `utils.unique` is used. I'm inclined to just use the Set constructor here and even down the road if needed again. <nit-pick>This will also remove the need to have a test for this function. </nit-pick>

```suggestion
  for (const url of new Set(linkedIssues)) {
```

Reply:
Cool! I'm still a bit new to typescript and didn't know about `Set`, I'm going to give this a go.

Reply:
It’s part of JavaScript. I don’t remember when it was introduced, ES6 maybe 🤔. IE unsupported for the most part but there’s no issues on Node.

Work pretty much as you would expect from other languages:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set